Makefile

```diff
@@ -1,2 +1,8 @@
+fresh: update build
 build:
-	go build -ldflags "-X main.APID_SOURCE_VERSION=`git rev-parse --abbrev-ref HEAD`-`git rev-parse HEAD`"
+	@(mv build_helper.go.1 build_helper.go; go build build_helper.go; mv build_helper.go build_helper.go.1)
```

Comment:
instead of namingit `*.go.1`, you can create a new folder called `build_helper`, and put `build_helper.go` into this folder. 
```
cd build_helper
go build
mv build_helper ../; cd  ..
./build_helper 2>builderr 1>buildapid; rm build_helper
```

Reply:
I was in double minds about this. Anyways, put them separate.